src/organisation/containers/organisation/organisation.component.html

```diff
@@ -26,16 +24,18 @@
     </tr>
     <!-- PBA Numbers -->
     <tr class="govuk-table__row">
-      <th class="govuk-table__header" scope="row">PBA number</th>
+      <th class="govuk-table__header" scope="row">PBA numbers</th>
       <td class="govuk-table__cell">
         <a class="govuk-link fright" routerLink="/organisation/update-pba-numbers" *ngIf="showChangePbaNumberLink" id="change-pba-account-numbers__link" aria-label="Change PBA account numbers">Change</a>
-
-        <div *ngIf="organisationPaymentAccount">
-          <div *ngFor="let pba of organisationPaymentAccount">{{pbaNumberWithStatus(pba)}}</div>
-        </div>
-
-        <div *ngIf="!organisationPaymentAccount">
-          <div>No PBA accounts registered</div>
+          <div *ngIf="organisationPaymentAccount">
```

Comment:
I think there's too much nesting here. Doesn't need the outer if statement as the ngFor will take care of the visibility.

Reply:
Remove unnecessary div

Reply:
Formatting of the div is a little out but I'll resolve for now.